web/src/helpers/formatting.js

```diff
@@ -16,38 +17,89 @@ const formatCo2 = function (d, numDigits) {
   d /= 1e6; // Convert to tCO₂ / min
   if (d == null || d === NaN) return d;
   if (numDigits == null) numDigits = 3;
-  if (d >= 1) // a ton or more
+  if (d >= 1)
+    // a ton or more
     return d3.format('.' + numDigits + 's')(d) + 't ' + translation.translate('ofCO2eqPerMinute');
-  else
-    return d3.format('.' + numDigits + 's')(d * 1e6) + 'g ' + translation.translate('ofCO2eqPerMinute');
+  else return d3.format('.' + numDigits + 's')(d * 1e6) + 'g ' + translation.translate('ofCO2eqPerMinute');
 };
 const scalePower = function (maxPower) {
   // Assume MW input
   if (maxPower < 1)
     return {
-      unit: "kW",
-      formattingFactor: 1e-3
-    }
+      unit: 'kW',
+      formattingFactor: 1e-3,
+    };
   if (maxPower < 1e3)
```

Comment:
Weird that this file wasn't prettified/linted earlier? 🤔 
And also weird that husky/prettier isn't taking effect here and fixing/complaining about lack of brackes in if-else statement...